src/halmos/mapper.py

```diff
@@ -0,0 +1,181 @@
+from dataclasses import dataclass
+from typing import Any, Dict, List, Optional, Type
+
+
+@dataclass
+class AstNode:
+    node_type: str
+    id: int
+    name: str
+    address: str
```

Comment:
naming: precisely speaking, this is not an address, but a selector ([4-byte signature](https://www.4byte.directory/)) for functions, events, or errors.

clarifying this or renaming it would be great.